language/German/strings.po

```diff
@@ -887,18 +893,24 @@ msgid ""
 "One or more ROMs/disk images for this system are incorrect. The system may "
 "not run correctly.\n"
 msgstr ""
+"Ein oder mehrere ROMs/Disk-Images für dieses System sind fehlerhaft. Das "
+"System läuft möglicherweise nicht richtig.\n"
 
 #: src/frontend/mame/ui/info.cpp:83
 msgid ""
 "There are known problems with this system\n"
 "\n"
 msgstr ""
+"Es gibt bekannte Probleme mit diesem System\n"
+"\n"
 
 #: src/frontend/mame/ui/info.cpp:88
 msgid ""
 "One or more ROMs/disk images for this system have not been correctly "
 "dumped.\n"
 msgstr ""
+"Ein oder mehrere ROMs/Disk-Images für dieses System wurden nicht korrekt "
+"gedumpt.\n"
 
```

Comment:
you force-translate "solid-state", but leave "gedumpt"? ;-)

Reply:
It's _ausgelesen_ now

Reply:
Is that the terminology the emulation community uses, though?

Reply:
true, but if we're translating "solid state media", "screenshot" and other well-known terms into German, "dumping" should also be translated? Then again I'm not sure who the target audience is for the German translation, i.e. what can you assume a random user knows and doesn't know. If I had done the translation I would have left many more english terms as-is

Reply:
It's also consistent with the other place in the file referring to _dumped_